tools/rlimit/rlimit.c

```diff
@@ -125,16 +129,26 @@ reapchild (int nsig)
 int
 main (int argc, char **argv)
 {
+  int begin;
   sigset_t block_cld;
 
   /* we need at least 3 args */
   if (argc < 3)
     usage ();
 
+  begin = 2;
```

Comment:
Weird logic ... Why 2 ? If you use 0, then you'll have:
`
/* argv[0] = .../rlimit
￼     argv[begin + 1] = seconds
￼     argv[begin + 2] = command
￼     argv[begin + 3] = args */
`
But that's just cosmetic ...
Still, I would not use that variable name. To me, something like `parsed_options`, or `argv_start_index` ... Again, cosmetic, but better for code reviewers ;)